eth/eventmonitor.go

```diff
@@ -75,30 +81,42 @@ func (em *eventMonitor) SubscribeNewRound(ctx context.Context, subName string, l
 		Topics:    [][]common.Hash{[]common.Hash{eventId}},
 	}
 
-	sub, err := em.backend.SubscribeFilterLogs(ctx, q, logsCh)
-	if err != nil {
-		return nil, err
+	subscribe := func() error {
+		sub, err := em.backend.SubscribeFilterLogs(ctx, q, logsCh)
+		if err != nil {
+			glog.Errorf("SubscribeNewRound error: %v. Retrying...", err)
+			return err
+		} else {
+			glog.Infof("SubscribeNewRound successful.")
```

Comment:
Is this a one-time message or would it lead to additional logging after each round?

Reply:
It's a one-time message per connection (it will re-print if there is a re-connection)